src/test/java/seedu/weme/testutil/TypicalMemes.java

```diff
@@ -20,33 +20,33 @@
 public class TypicalMemes {
 
     public static final Meme ALICE = new MemeBuilder().withName("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111")
+            .withDescription("123, Jurong West Ave 6, #08-111")
```

Comment:
Maybe it's better to create stub `description`s that are more relevant to memes?